pallets/loans/src/migrations.rs

```diff
@@ -56,25 +63,29 @@ pub mod v2 {
 
     /// Migration to sorted [`SortedListProvider`].
     pub fn migrate<T: Config>() -> Weight {
-        if StorageVersion::<T>::get() == crate::Versions::V1 {
-            log::info!("migrating loans to Versions::V2",);
+        if StorageVersion::<T>::get() == crate::Versions::V2 {
+            log::info!("migrating loans to Versions::V3",);
 
-            Markets::<T>::translate::<OldMarket<BalanceOf<T>>, _>(|_key, market| {
+            Markets::<T>::translate::<V2Market<BalanceOf<T>>, _>(|_key, market| {
                 Some(Market {
-                    borrow_cap: 1_000_000_000_000_000u128,
-                    supply_cap: market.cap,
+                    borrow_cap: market.borrow_cap,
+                    supply_cap: market.supply_cap,
                     collateral_factor: market.collateral_factor,
+                    liquidation_threshold: (market.collateral_factor
+                        + market.collateral_factor * DEFAULT_LIQUIDATE_THRESHOLD),
```

Comment:
Let's add 10% here. 
DEFAULT_LIQUIDATE_THRESHOLD can be defined in migrations.rs, and I think DEFAULT_LIQUIDATION_OFFSET is better.